include/deal.II/base/function_lib.h

```diff
@@ -930,25 +935,71 @@ namespace Functions
      *
      * If an argument <tt>select</tt> is given and not -1, the cut-off
      * function will be non-zero for this component only.
+     *
+     * If the argument @p integrate_to_one is set to true, then the value of
+     * the function is rescaled whenever a new radius is set.
      */
     CutOffFunctionBase(
       const double radius             = 1.,
       const Point<dim>                = Point<dim>(),
       const unsigned int n_components = 1,
-      const unsigned int select       = CutOffFunctionBase<dim>::no_component);
+      const unsigned int select       = CutOffFunctionBase<dim>::no_component,
+      const bool         integrate_to_one       = false,
+      const double       unitary_integral_value = 1.0);
```

Comment:
can you also document the last argument? I'm unsure what the term "unitary" in it means, but it would only be unitary if the corresponding value were 1.0, so I'm a bit confused :-)